Sources/Brave/Frontend/Rewards/BraveRewards.swift

```diff
@@ -228,17 +228,6 @@ public class BraveRewards: NSObject {
     )
   }
 
-  /// Report posting data to a form
-  func reportPostData(_ data: Data, url: URL, tabId: Int, firstPartyURL: URL, referrerURL: URL?) {
```

Comment:
@kylehickinson to confirm Rewards changes

Reply:
Hah, I was the one who commited this 🙂

Reply:
right, i did not the commit history!